packages/subgraph/package.json

```
@@ -4,14 +4,15 @@
   "scripts": {
     "prepare": "./scripts/prepare.sh",
     "codegen": "yarn && graph codegen",
-    "test": "yarn codegen && graph test",
+    "test": "yarn codegen && yarn prep:test && graph test",
     "build": "yarn && yarn prepare && graph build",
     "deploy": "yarn codegen && yarn prep:prod && graph deploy --node https://api.thegraph.com/deploy/ edgeandnode/block-oracle",
     "create-local": "graph create --node http://127.0.0.1:8020/ edgeandnode/block-oracle",
     "remove-local": "graph remove --node http://127.0.0.1:8020/ edgeandnode/block-oracle",
-    "deploy-local": "yarn codegen && graph deploy --node http://127.0.0.1:8020/ --ipfs http://localhost:${IPFS_PORT} edgeandnode/block-oracle --version-label 0.1.0",
-    "prep:local": "mustache ./config/local.json subgraph.template.yaml > subgraph.yaml",
-    "prep:prod": "mustache ./config/prod.json subgraph.template.yaml > subgraph.yaml"
+    "deploy-local": "yarn codegen && yarn prep:local && graph deploy --node http://127.0.0.1:8020/ --ipfs http://localhost:${IPFS_PORT} edgeandnode/block-oracle --version-label 0.1.0",
```

This call to `prep:local` is overwriting the changes introduced by `graph build --network hardhat`. Could we remove it?

```suggestion
    "deploy-local": "yarn codegen && graph deploy --node http://127.0.0.1:8020/ --ipfs http://localhost:${IPFS_PORT} edgeandnode/block-oracle --version-label 0.1.0",
```

---

I see that this is fixed in 35bcfec17dded7f1f3fab70196a547e30d73029c. Thanks!